packages/expo-constants/src/__tests__/Constants-test.ts

```diff
@@ -79,12 +79,15 @@ describe(`manifest`, () => {
       const UnimodulesCore = jest.requireActual('expo-modules-core');
       return {
         ...UnimodulesCore,
-        NativeModulesProxy: {
-          ...(UnimodulesCore.NativeModulesProxy ?? {}),
-          ExpoUpdates: {
-            ...(UnimodulesCore.NativeModulesProxy?.ExpoUpdates ?? {}),
+        requireOptionalNativeModule: (moduleName) => {
```

Comment:
nit: method syntax works as well and is a little shorter
```suggestion
        requireOptionalNativeModule(moduleName) {
```